runtime/themes/emacs.toml

```diff
@@ -0,0 +1,636 @@
+# Ported by Yevgnen.
+
+"comment" = { fg = "firebrick" }
+"constant" = { fg = "dark_cyan" }
+"constant.builtin" = { fg = "dark_slate_blue" }
+"function" = { fg = "blue1" }
+"function.call" = { fg = "blue1" }
+"function.macro" = { fg = "dark_slate_blue" }
+"function.builtin" = { fg = "dark_slate_blue" }
+"function.special" = { fg = "dark_slate_blue" }
+"keyword" = { fg = "purple" }
+"operator" = { fg = "purple" }
+"punctuation" = { fg = "black" }
+"string" = { fg = "violetred4" }
+"string.special" = { fg = "violetred4", modifiers = ["bold"] }
+"type" = { fg = "forest_green" }
+"type.builtin" = { fg = "dark_slate_blue" }
+"constructor" = { fg = "forest_green" }
+"variable" = { fg = "sienna" }
+"variable.builtin" = { fg = "dark_slate_blue" }
+"variable.parameter" = { fg = "sienna" }
+"tag" = { fg = "blue1" }
+"label" = { fg = "dark_slate_blue" }
+"attribute" = { fg = "dark_slate_blue" }
+"namespace" = { fg = "dark_cyan" }
+"module" = { fg = "dark_cyan" }
+"special" = { fg = "blue1" }
+
+"markup.heading" = { fg = "gold", modifiers = ["bold"] }
+"markup.heading.marker" = { fg = "gray60" }
+"markup.heading.1" = { fg = "blue1" }
+"markup.heading.2" = { fg = "sienna" }
+"markup.heading.3" = { fg = "purple" }
+"markup.heading.4" = { fg = "firebrick" }
+"markup.heading.5" = { fg = "forest_green" }
+"markup.heading.6" = { fg = "dark_cyan" }
+"markup.list" = { fg = "black" }
+"markup.bold" = { modifiers = ["bold"] }
+"markup.italic" = { modifiers = ["italic"] }
+"markup.link.url" = { fg = "royalblue3", modifiers = ["underlined"] }
+"markup.link.text" = { fg = "royalblue3" }
+"markup.quote" = { fg = "gray60" }
+"markup.raw.inline" = { fg = "dark_cyan" }
+"markup.raw.block" = { fg = "dark_cyan" }
+
+"ui.background" = { fg = "black", bg = "white" }
+"ui.background.separator" = { fg = "black", bg = "white" }
+"ui.cursor" =  { fg = "white", bg = "gray70" }
+"ui.cursor.primary" = { fg = "white", bg = "black" }
+"ui.cursor.match" = { fg = "black", bg = "turquoise" }
+"ui.cursor.select" = { fg = "white", bg = "black" }
+"ui.cursor.insert" = { fg = "red", bg = "black" }
+"ui.linenr" = { fg = "gray60" }
+"ui.linenr.selected" = { fg = "gray80" }
+"ui.statusline" = { fg = "black", bg = "gray75" }
+"ui.statusline.inactive" = { fg = "gray20", bg = "gray90" }
+"ui.popup" = { fg = "black", bg = "gray97" }
+"ui.popup.info" = { fg = "black", bg = "gray97" }
+"ui.window" = { fg = "black" }
+"ui.help" = { fg = "dark_red", bg = "cornsilk" }
+"ui.text" = { fg = "black" }
+"ui.text.focus" = { fg = "black", bg = "darkseagreen2" }
+"ui.menu" = { fg = "black", bg = "cornsilk" }
+"ui.menu.selected" = { fg = "dark_red", bg = "light_blue" }
+"ui.selection" = { bg = "lightgoldenrod1" }
+"ui.selection.primary" = { bg = "lightgoldenrod2" }
+"ui.virtual.whitespace" = "highlight"
+"ui.virtual.ruler" = { bg ="gray95" }
+
+# Disabled by default
+# "ui.cursorline.primary" = { bg = "darkseagreen2" }
+# "ui.cursorline.secondary" = { bg = "darkseagreen2" }
```

Comment:
These are behind a configuration option so they don't need to be commented out to not use them

Reply:
Done!